templates/zerver/register.html

```diff
@@ -90,6 +91,34 @@ <h1>You&rsquo;re almost there.</h1>
                 </div>
                 {% endif %}
 
+                {% if user_avatar_url %}
+                <hr>
+                <div class="user-avatar-box">
+                    <div id="user-avatar-upload-widget-portico" class="avatar-icon-logo-settings">
+                        <div class="image_upload_button">
+                            <div class="settings-page-image-background"></div>
```

Comment:
I think we can use `image_upload_widget.hbs` and `image_upload_widget.scss`  instead of writing it again. like we have already used it for `user-avatar`, 
`eg something like this`
```
<div id="user-avatar-upload-widget-portico" class="image_upload_widget">
{{> image_upload_widget
      widget = "user-avatar-upload-widget-portico"
      upload_text = (t "Upload profile picture")
      delete_text = (t "Delete profile picture")
      is_editable_by_current_user = true
      image = page_params.avatar_url_medium
}}
<div id="user-avatar-source">
      <a href="https://en.gravatar.com/" target="_blank">{{ _("Avatar from Gravatar") }}</a>
 </div>
</div>
```

Reply:
Yeah, I tried to import it, but as this is not a `.hbs` file and instead is a `.html` I could not import it as you just showed. Also, it has some extra `input` tags that I use in the backend to check the things like `avatar source` and if the user is choosing or not to use their Github avatar instead or Gravatar or an uploaded picture.

Reply:
https://stackoverflow.com/questions/25758144/include-a-handlebars-or-a-hbs-file-in-html I think we can use `script` tag

Reply:
and also you can use available CSS in `image_upload_widget.sccs` . you are some `old css ` like `settings-page-image-background` .

Reply:
No, that's just because this branch isn't rebased. My local branch is with the updated CSS but I'm doing a bunch of changes that only when it works I'll push it here in order to not lose all I have done so far that is currently working.

Reply:
ok great

Reply:
I should have updated this PR, isn't ready for review now, but it'll be by the end of the day